src/remembrall/helpers/book_operations.py

```diff
@@ -136,21 +135,47 @@ def change_email(args: list, book: AddressBook) -> str:
 
 @input_error
 def add_address(args: list, book: AddressBook) -> str:
-    name, address = args
+    '''Add address to a contact. If contact does not exist, return "Contact not found."
+    If address is already present, return "Address already exists."
+    If address is added, return "Address added."
+    '''
+    if len(args) < 1:
+        raise IndexError()
+    name, address = ' '.join(args).split(' ', 1)
+    if not address:
+        raise ValueError()
     found = book.find(name)
     if not found:
         return "Contact not found."
-    # TODO: Add address to record
-    return "Address added"
+    return found.add_address(address) 
```

Comment:
в цій функції ми ж повинні вивести повідомлення, тому клаще так
```suggestion
    found.add_address(address) 
    return "Address added"
```

Reply:
Зрозумів, зробив